sr_multi_description/urdf/bimanual_srhand_ur.urdf.xacro

```diff
@@ -61,23 +62,31 @@
     <xacro:include filename="$(find sr_description)/hand/xacro/hand.urdf.xacro"/>
     <xacro:include filename="$(find sr_description)/other/xacro/gazebo.urdf.xacro"/>
 
-    <xacro:sr_hand side="right" hand_type="$(arg right_hand_type)" hand_version="$(arg right_hand_version)"
-                   fingers="$(arg right_fingers)" tip_sensors="$(arg right_tip_sensors)" mid_sensors="$(arg right_mid_sensors)"
-                   prox_sensors="$(arg right_prox_sensors)" palm_sensor="$(arg right_palm_sensor)"/>
+    <xacro:if value="${'$(arg hands)' == 'both' or '$(arg hands)' == 'right'}">
+        <xacro:sr_hand side="right" hand_type="$(arg right_hand_type)" hand_version="$(arg right_hand_version)"
+                    fingers="$(arg right_fingers)" tip_sensors="$(arg right_tip_sensors)" mid_sensors="$(arg right_mid_sensors)"
+                    prox_sensors="$(arg right_prox_sensors)" palm_sensor="$(arg right_palm_sensor)"/>
+    </xacro:if>
 
-    <joint name="ra_arm_to_hand" type="fixed">
-        <parent link="ra_flange"/>
-        <child link="rh_forearm"/>
-        <origin xyz="0.012 0.0 0.0" rpy="${pi/2.0} ${pi} ${pi/2.0}"/>
-    </joint>
+    <xacro:if value="${'$(arg hands)' == 'both' or '$(arg hands)' == 'right'}">
```

Comment:
As both if are the same here and in the section below I would group them into the same if statement